Assets/HoloToolkit/Utilities/Scripts/WorldAnchorManager.cs

```diff
@@ -136,6 +137,36 @@ public void RemoveAnchor(GameObject gameObjectToUnanchor)
         }
 
         /// <summary>
+        /// Removes all anchors from the scene and deletes them from the anchor store.
+        /// </summary>
+        public void RemoveAllAnchors()
+        {
+            // This case is unexpected, but just in case.
+            if (AnchorStore == null)
+            {
+                Debug.LogError("remove all anchors called before anchor store is ready.");
+            }
+
+            WorldAnchor[] anchors = FindObjectsOfType<WorldAnchor>();
```

Comment:
Benefits of storing all user anchors may not be relevant, if it's just for one RemoveAll method.
At this time, I am not prioritizing this anchor array